lib/moxiedoc.js

```diff
@@ -2,7 +2,7 @@ var Parser = require('./parser').Parser;
 var Builder = require('./builder').Builder;
 var Exporter = require('./exporter').Exporter;
 var reporter = require('./reporter');
-var wildcard = require('wildcard');
+var matcher = require('matcher');
```

Comment:
I assume there was an issue with the `wildcard` library?

Reply:
https://www.npmjs.com/package/wildcard
> NOTE
> Work on this project is largely inactive, now so I'd recommend checking out the wonderful matcher package as a solid alternative.

no problem, just moving to an actively maintained alternative